client/lib/feed-post-store/actions.js

```diff
@@ -102,4 +104,7 @@ FeedPostActions = {
 	},
 };
 
+const { fetchPost, receivePost, markSeen } = FeedPostActions;
+export { fetchPost, receivePost, markSeen };
```

Comment:
why this and not…

```js
export const { fetchPost, receivePost, markSeen } = FeedPostActions;
```

is this a result of the codemod not recognizing that they could be combined?

Reply:
i don't remember if it was as a result of me or a codemod at this point. its been awhile.